asyncpgsa/pool.py

```diff
@@ -5,10 +5,16 @@
 
 
 class SAPool(Pool):
+    __slots__ = ('_debug',)
```

Comment:
I dont think this slots is compatible with `Pool` slots. This might break things. If slots are incompatible I dont think you can do inheritance. Might complicate things.